### PR TITLE
Raise error when description can't be analyzed

### PR DESCRIPTION
**Pull request by cornelius**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 3d5b855fa90c7833655967c9a3f95277da865958 into 175cb6b349dff2b34faa6e9863613edc828ba502 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 3d5b855fa90c7833655967c9a3f95277da865958 into 070b66eb20efd1514313c5c84866a2c827f1a43a )

File: lib/analyze_config_file_diffs_task.rb [hunk 1]
```diff
@@ -17,10 +17,11 @@
 
 class AnalyzeConfigFileDiffsTask
   def analyze(description)
+    description.assert_scopes("os")
+    description.validate_analysis_compatibility
     description.assert_scopes(
       "repositories",
       "config_files",
```
Comment (thardeck) [inline lib/analyze_config_file_diffs_task.rb]: the comma needs to be removed I guess

Comment (cornelius) [inline lib/analyze_config_file_diffs_task.rb]: It works with the comma as well, but you are right, of course. Will remove it.

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 6a2b715f0d53fc9f67bd94b7ca92e871ca9b4f36 into 070b66eb20efd1514313c5c84866a2c827f1a43a )

**Comment by thardeck** [comment]
besides that lgtm

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged ead6bc7b559066d42bb934b2468476ca81dcd8b2 into 070b66eb20efd1514313c5c84866a2c827f1a43a )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 40eae72850e663aa46a87336d9f5306c610046f0 into 070b66eb20efd1514313c5c84866a2c827f1a43a )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 40eae72850e663aa46a87336d9f5306c610046f0 into 99cd2f3abde658ee652bb2c39a4f311440cc5801 )

**Comment by thardeck** [comment]
lgtm besides my comment

**Comment by cornelius** [comment]
Obsoleted by #551.

**Pull request closed without merging**